<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKeyList</class>
 <widget class="QWidget" name="FormKeyList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButtonAdd">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxNew">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButtonRemove">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeView" name="treeViewList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../DbUi.qrc">
     <normaloff>:/Icons/Menu New.png</normaloff>:/Icons/Menu New.png</iconset>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="toolTip">
    <string>Add new</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../DbUi.qrc">
     <normaloff>:/Icons/Menu Remove.png</normaloff>:/Icons/Menu Remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove selected</string>
   </property>
   <property name="toolTip">
    <string>Remove selected</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../DbUi.qrc"/>
 </resources>
 <connections/>
</ui>
